### PR TITLE
.org file for configuring kind+auditsink

### PR DESCRIPTION
**Pull request by hh**
We lost the dynamic audit sink configuration in 1.19.
We also don't usually build from src, but it makes sense to.

These directions will help, and we will likely want to update our humans image, and the directions to build/use this in another PR.

### CODE REVIEW

File: org/kind+auditsink.org [hunk 1]
```diff
@@ -0,0 +1,797 @@
...
+469817root      0:00 /bin/sh
+470159root      0:00 /bin/sh
+470160root      0:00 bash
+470161root      0:00 ps ax
+#+end_src
+
+* Build kind image
+** go get k8s
+#+begin_src shell :async yes
+  go get -u github.com/kubernetes/kubernetes
```
Comment (heyste) [inline org/kind+auditsink.org:546]: From this [documentation](https://github.com/kubernetes/kubernetes#you-have-a-working-go-environment) it looks like the following is required
```
mkdir -p $GOPATH/src/k8s.io
cd $GOPATH/src/k8s.io
```

Comment (heyste) [reply org/kind+auditsink.org:546]: then switch `go get -u github.com/kubernetes/kubernetes`
for `git clone https://github.com/kubernetes/kubernetes`

File: org/kind+auditsink.org [hunk 2]
```diff
@@ -0,0 +1,797 @@
...
+#+end_src
+** ensure k8s is updated
+#+begin_src shell :async yes
+  cd `go env GOPATH`/src/github.com/kubernetes/kubernetes
```
Comment (heyste) [inline org/kind+auditsink.org:550]: update to ```cd `go env GOPATH`/src/k8s.io/kubernetes```

File: org/kind+auditsink.org [hunk 3]
```diff
@@ -0,0 +1,797 @@
...
+  git remote -v origin
```
Comment (heyste) [inline org/kind+auditsink.org:551]: is the `origin` at the end a typo?

File: org/kind+auditsink.org [hunk 4]
```diff
@@ -0,0 +1,797 @@
...
+nothing to commit, working tree clean
+#+end_example
+
+** kind build
+
+This can take a while, and you can allocate more CPU and memory to your docker VM to speed things up.
+
+#+begin_src tmate
+  ls -la ~/go/src/github.com/kubernetes/kubernetes
+  kind build node-image --kube-root ~/go/src/github.com/kubernetes/kubernetes
```
Comment (heyste) [inline org/kind+auditsink.org:570]: Using the updated path
```
ls -la ~/go/src/k8s.io/kubernetes
kind build node-image --kube-root ~/go/src/k8s.io/kubernetes
```

**Comment by BobyMCbobs** [comment]
Thank you @heyste for your suggestions, the changes have been made accordingly

**Pull request merged**